src/doc/book/ownership.md

```diff
@@ -51,15 +51,24 @@ fn foo() {
 }
 ```
 
-When `v` comes into scope, a new [`Vec<T>`][vect] is created. In this case, the
-vector also allocates space on [the heap][heap], for the three elements. When
-`v` goes out of scope at the end of `foo()`, Rust will clean up everything
-related to the vector, even the heap-allocated memory. This happens
-deterministically, at the end of the scope.
+When `v` comes into scope, a new [vector][] is created, and it allocates space
```

Comment:
Was going to blame my preferred markdown tool for not supporting this syntax, turns out [it does](http://pandoc.org/README.html#extension-shortcut_reference_links)! Today I learned. Will fix.